loadgen/loadgen.cc

```diff
@@ -1492,7 +1492,26 @@ void StartTest(SystemUnderTest* sut, QuerySampleLibrary* qsl,
     detail("*TestSettings (performance_sample_count_override) can override");
     detail("*Refer to Effective Settings for actual value");
   });
-  loadgen::TestSettingsInternal sanitized_settings(requested_settings, qsl);
+
+  TestSettings test_settings = requested_settings;
+  // Look for Audit Config file to override TestSettings during audit
+  std::ifstream audit_config("audit.config");
+  if (audit_config.good()) {
+    LogDetail([] (AsyncDetail& detail) {
+      detail("Found Audit Config file (audit.config)."
+              " Overriding TestSettings from audit.config file.");
+    });
+    std::string audit_scenario = loadgen::ToString(test_settings.scenario);
+    // Remove Spaces from the string
+    audit_scenario.erase(remove(audit_scenario.begin(), audit_scenario.end(),
```

Comment:
In util.h, there's a RemoveValue you could use:
RemoveValue(&audit_scenario, ' ');